cmds/live.c

```diff
@@ -13,14 +13,13 @@
 #include "utils/socket.h"
 #include "utils/utils.h"
 
-static char *tmp_dirname;
+#define TMP_DIR_NAME_SIZE 32
+
+static char tmp_dirname[TMP_DIR_NAME_SIZE] = { '\0', };
 static void cleanup_tempdir(void)
 {
-	if (!tmp_dirname)
-		return;
-
 	remove_directory(tmp_dirname);
```

Comment:
I think wrong `dirname` would be covered with `opendir()` cheking.

Reply:
But let's check if `tmp_dirname` has a valid string first.

Reply:
I think it's ok to check the first byte only.

```c
if (tmp_dirname[0] == '\0')
    return;
```

Reply:
I agree, Done!